### PR TITLE
add posts in 2019

### PR DESCRIPTION
**Pull request by dhmlau**
Signed-off-by: Diana Lau <dhmlau@ca.ibm.com>

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ const darkCodeTheme = require('prism-react-renderer/themes/dracula');
 const config = {
   title: 'LoopBack',
   tagline: 'Dinosaurs are cool',
-  url: 'https://loopbackio.github.io',
+  url: 'https://blog.loopback.io',
```
Comment (achrinza) [inline docusaurus.config.js:15]: Non-blocker; Are we planning to use the `blog` subdomain, or still going forward with <https://loopback.io/blog>? I'm currently writing a shell script for to vendor this repository into <https://github.com/loopbackio/loopback.io>. Still working out the kinks, but will open a PR there once it's ready.

**Review by achrinza — APPROVED** [review]
LGTM :+1: 

I noticed that - on this branch and the `main` branch - `npm build` does not work while `npm start` (i.e. live server) works fine. Will need to investigate further.

Also just a minor comment :point_down:

File: docusaurus.config.js (hunk 1 above)
Comment (dhmlau) [reply docusaurus.config.js:15]: @achrinza, I cannot find a way to have GitHub pages from 2 repos using the same domain, so subdomain `blog.loopback.io` is being used. If you find a way to do it, I can update it as well.

Comment (achrinza) [reply docusaurus.config.js]: This has been resolved in https://github.com/loopbackio/loopback-blog/pull/9

**Pull request merged**